lightning/src/ln/channel.rs

```diff
@@ -2969,6 +2969,30 @@ impl<Signer: Sign> Channel<Signer> {
 			panic!("Cannot update fee while peer is disconnected/we're awaiting a monitor update (ChannelManager should have caught this)");
 		}
 
+		// Before to propose a feerate update, check that we can actually afford the new fee compare to the holder balance.
+		if let Some(reserve) = self.counterparty_selected_channel_reserve_satoshis {
```

Comment:
If we're `is_usable` we *definitely* have filled in the channel reserve value :)